### ISSUE TITLE
typo in the setup guide for new project

### ISSUE THREAD

**Issue by ThaYoost** [electro-smith/DaisyWiki#44]
Great setup guide, it's worked like a charm (on PC). Only small thing I saw:
At https://github.com/electro-smith/DaisyWiki/wiki/How-To-Create-a-New-Project
in step 4 it says  to refer to step "3" of the dev env guide. That should be "4"

Also, in setting up the dev env the first time I needed to "Build all" it did not work because there were some "makefile extensions that weren't installed yet. I missed the pop-up the first time through. 

going back to learning DSP now :D

**Comment by crosswick** [comment]
Yes I also found a typo: https://electro-smith.github.io/libDaisy/index.html#autotoc_md71

The Creating a New Project link has an extra colon in it.

**Comment by andrewikenberry** [comment]
These have been fixed and are live on the new support site.
daisy.audio
